syntax = "proto3";

package iris;

message IrisPredictRequest {
    float sepal_length = 1;
    float sepal_width = 2;
    float petal_length = 3;
    float petal_width = 4;
}

message IrisPredictResponse {
    string target = 1;
}

service Predict{
    rpc predict_iris_target(IrisPredictRequest) returns (IrisPredictResponse){}
}